packages/theming/atlas/src/theme/native/main.ts

```diff
@@ -0,0 +1 @@
+import {} from "../../themesource/atlas_ui_resources/native/core/variables";
```

Comment:
or should we import custom-variables? Not sure myself

Reply:
In default Atlas, it doesn't matter. But users may cleanup / remove certain variables, which makes `variables` less error prone. On the other hand, people can add new variables to the `custom-variables`, which don't exist in `variables`.
I've changed the exports in `variables` to below, that should do the trick right?
```
export * from "../../../../theme/native/custom-variables";
export {
    brand,
    background,
    border,
    button,
    contrast,
    font,
    input,
    image,
    layoutGrid,
    listView,
    navigation,
    spacing,
    container,
    tabContainer,
    badge,
    floatingActionButton,
    introScreen,
    listViewSwipe,
    progressBar,
    progressCircle,
    slider,
    rating
};
```

Reply:
But this way you override custom-variables with the default values, while other way around would be more logical, wouldn't it?

Reply:
Yes, but I should have included the whole file to give you the right picture. We are merging custom variables with default variables above the exports. Example:
```
import * as custom from "../../../../theme/native/custom-variables";

let rating: VariablesRating = {
    containerDisabled: {
        opacity: 0.5
    },
    icon: {
        size: 24,
        color: contrast.lower,
        selectedColor: brand.warning
    }
};
rating = merge(rating, custom.rating || ({} as any));

export * from "../../../../theme/native/custom-variables";
export {
    rating
};
```

Tested and working 😄